src/main/java/net/dries007/tfc/common/entities/predator/Cougar.java

```diff
@@ -0,0 +1,85 @@
+package net.dries007.tfc.common.entities.predator;
+
+import net.dries007.tfc.client.RenderHelpers;
+import net.dries007.tfc.common.entities.aquatic.Jellyfish;
+import net.minecraft.nbt.CompoundTag;
+import net.minecraft.nbt.Tag;
+import net.minecraft.network.syncher.EntityDataAccessor;
+import net.minecraft.network.syncher.EntityDataSerializers;
+import net.minecraft.network.syncher.SynchedEntityData;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.DifficultyInstance;
+import net.minecraft.world.entity.EntityType;
+import net.minecraft.world.entity.MobSpawnType;
+import net.minecraft.world.entity.SpawnGroupData;
+import net.minecraft.world.level.Level;
+import net.minecraft.world.level.ServerLevelAccessor;
+import org.jetbrains.annotations.Nullable;
+
+public class Cougar extends BigCat {
```

Comment:
Black panthers should be a separate entity type rather than this special implementation, that's why we have the ability to do the `createDiurnal` thing and all that to make instantiation easy